js/types.ts

```diff
@@ -503,7 +503,7 @@ export interface AppState {
 
 export interface Extras {
   requireJSZip: () => Promise<never>;
-  requireJSMediaTags: () => Promise<never>;
+  requireMusicMetadata: () => Promise<never>;
```

Comment:
Since you have real types for music-metadata, could we actually type the promised value?